apps/docs/content/components/SectionIntro.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: SectionIntro
+status: Experimental
+source: https://github.com/primer/brand/tree/main/src/SectionIntro/SectionIntro.tsx
+storybook: '/brand/storybook/?path=/story/components-SectionIntro--playground'
+description: Use SectionIntro to provide a title and optional description paragraph and action to a new section in the page.
+---
+
+import {Label} from '@primer/react'
+
+```js
+import {SectionIntro} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
```

Comment:
@danielguillan - Given that this is now rendering as a `header`, can we put a Warning Flash message here to mention that `SectionIntro` should be wrapped in a `section` please? That or our examples should include `section` tags in the code snippets.